### PR TITLE
Raise row/window heights in various places

### PR DESCRIPTION
**Pull request by AaronVanGeffen**
Back with some more row/window heights changes to better accommodate CJK languages in various windows.

This PR tackles:
* Staff window (stats tab)
* Ride list window (scrollable list)
* Track list window (scrollable list, stats listing)
* Park window (stats, objective)
* Map window (legend)

### CODE REVIEW

File: src/openrct2-ui/windows/Map.cpp [hunk 1]
```diff
@@ -860,13 +863,15 @@ static void window_map_paint(rct_window *w, rct_drawpixelinfo *dpi)
                 STR_MAP_TOILET,
             };
 
-            for (sint32 i = 0; i < 8; i++) {
+            for (sint32 i = 0; i < 8; i++)
```
Comment (Gymnasiast) [inline src/openrct2-ui/windows/Map.cpp]: Can you replace this with `Util::CountOf(mapLabels)`? You'll probably need to change sint32 accordingly, seeing as CountOf return an unsigned integer.

**Comment by Gymnasiast** [comment]
Tested it. Works as intended.
Codewise, I have one request, which I added as a line note.

File: src/openrct2-ui/windows/Map.cpp (hunk 1 above)
Comment (AaronVanGeffen) [reply src/openrct2-ui/windows/Map.cpp]: Hmm, you mean a range-based for-loop? The index `i` is used for both `RideKeyColours` and `mapLabels`, though, so I think sticking to the index-based approach is better.

Comment (Gymnasiast) [reply src/openrct2-ui/windows/Map.cpp]: In that case, please use an Util::CountOf, rather than a hard coded number. Also make `i` unsigned.

Comment (AaronVanGeffen) [reply src/openrct2-ui/windows/Map.cpp]: Sure, that works. Done.

**Pull request merged**